ci-operator/config/red-hat-storage/odf-console/red-hat-storage-odf-console-master.yaml

```diff
@@ -0,0 +1,19 @@
+build_root:
+  project_image:
+    dockerfile_path: Dockerfile.ci
+resources:
+  "*":
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+  - as: lint
+    commands: yarn run lint
+    container:
+      from: root
```

Comment:
`root` does not have your source code injected (which is I assume the thing you want to lint)

Reply:
would src have it?

Reply:
I can only link to https://docs.ci.openshift.org/docs/architecture/ci-operator/#building-artifacts that describes what image contains what.